dmgbuild/core.py

```diff
@@ -378,7 +378,7 @@ def build_dmg(filename, volume_name, settings_file=None, settings={},
         # size of the background image, volume icon, and .DS_Store file (and
         # 128 MB should be well sufficient for even the most outlandish image
         # sizes, like an uncompressed 5K multi-resolution TIFF)
-        total_size = 128 * 1024 * 1024
+        total_size = 128e6
```

Comment:
I think this line should stay in base-2 units.

Reply:
will revert... do you want everything but the last conversion to stay base 2?

Reply:
Yes, I think that's the right approach here. Sorry :-)

Reply:
no problem!  I can't say I fully understand when we are and when we aren't in base-10, so happy to follow your lead there!

Reply:
FWIW, the reasoning is that file sizes need to be rounded up to a multiple of the filesystem block size (4096 bytes), so that we can calculate the number of bytes we're actually going to use. The base-10 part comes where we pass that value as a number of KB to hdiutil, which apparently accepts base-10 numbers these days.

Reply:
thanks!  good to know